### ISSUE TITLE
Fullscreen doesn't hide window titlebar (GNOME)

### ISSUE THREAD

**Issue by deflagrator** [harmonoid/harmonoid#399]
When opening the now playing screen in full screen mode, the window titlebar is still visible. Fullscreen ui would look a lot better without it.

![Screenshot from 2023-02-03 14-55-30](https://user-images.githubusercontent.com/96728391/216566566-a545e731-92eb-4b2d-8396-48579b5e9fac.png)

**Comment by alexmercerind** [comment]
I'm not super sure about this, but that's not how it should look like. I'm not on GNOME currently.

There was #380 issue which was resolved in last update. Are you on latest version?

**Comment by tactical-snail** [comment]
Yeah I'm on the latest version 0.3.5. Actually that titlebar in harmonoid doesn't respect the system dark theme either. It gets dark only if you set legacy applications to follow adwaita-dark theme from gnome tweak tool.

Also, check the latest reply on #380. It is still there.

And just to clarify, I'm talking about hiding the titlebar when entering fullscreen mode by clicking the icon in the bottom right corner of the now-playing screen. The previous issue seems to mention hiding it when maximizing the window.

**Comment by alexmercerind** [comment]
Try out the latest update please.
